share/functions/funced.fish

```diff
@@ -83,10 +83,16 @@ function funced --description 'Edit function definition'
     or return 1
     set -l tmpname $tmpdir/$funcname.fish
 
-    if functions -q -- $funcname
-        functions -- $funcname >$tmpname
-    else
+    set -l writepath
+
+    if not functions -q -- $funcname
         echo $init >$tmpname
+    else if functions --details -- $funcname | string match --invert --regex '^(?:-|stdin)$'
```

Comment:
This causes it to output the file path to stdout. I think we should add `--quiet` as a flag.